templates/flagger-loadtester.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: flagger-loadtester
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: flagger-loadtester
+  annotations:
```

Comment:
We need to add the `catalog.kubeaddons.mesosphere.io/addon-revision` annotation to both of these addons to ensure compatibility with kubeaddons-catalog. Later this should be covered by CI.

Documentation can be found at https://github.com/mesosphere/kubeaddons/tree/master/tools/catalog/README.md.